dicoogle/src/main/java/pt/ua/dicoogle/server/DicomStorage.java

```diff
@@ -78,6 +78,7 @@
     private BlockingQueue<ImageElement> queue = new PriorityBlockingQueue<>();
     private NetworkApplicationEntity[] naeArr = null;
     private AtomicLong seqNum = new AtomicLong(0L);
+    private static boolean ASYNC_INDEX = Boolean.valueOf(System.getProperty("dicoogle.index.async", "true"));

```

Comment:
We can add it later, but the server settings are much nicer to expand in v3, so we can add this setting there as well, leaving the environment variable as an override.

Reply:
We can think about it later.

Reply:
#602